engine/spec/spec.go

```diff
@@ -13,6 +13,7 @@ type (
 		Volumes  []*Volume         `json:"volumes,omitempty"`
 		Network  Network           `json:"network"`
 		Envs     map[string]string `json:"envs,omitempty"`
+		Files    []*File           `json:"files,omitempty"`
```

Comment:
File should not be a part of engine spec ( only needs to be present in api section).

Reply:
You are right. But I'd leave here too. Because then we will know what files we created if it's ever needed later.